VBA/Excel-VBA/articles/excel-performance-and-limit-improvements.md

```diff
@@ -1,247 +1,165 @@
-
-[Excel 2010 Performance Improvements](#xl2010PerfImp)
-
-[Conclusion](#office2016excelperf_Conclusion)
-
-[About the Author](#xlAboutAuthors)
-
-[Additional Resources](#office2007excelperf_AdditionalResources)
-
-**Provided by:** ![MVP Contributor](images/mvp_avatar.jpg) Charles Williams,  [Decision Models Limited](http://www.decisionmodels.com/) | [About the Author](excel-improving-calcuation-performance.md#xlAboutAuthor)
-
-
-## Excel 2016 Performance Improvements
+## Excel 2016 performance improvements
 <a name="xl2016PerfImp"> </a>
-Microsoft Office has fundamentally changed its development and release methodology with Office 16 and Office 365. Updates and new features are released on a regular cycle, so it becomes important to note the build number that improvements were released, and to check which build number you are using.The timescale that updates become available to you depends on which update option you are using:
+
+Microsoft Office has fundamentally changed its development and release methodology with Office 2016 and Office 365. Updates and new features are released on a regular cycle, so it becomes important to note the build number that improvements were released, and to check which build number you are using. The timescale that updates become available to you depends on which update option you are using:
 - Insider
 - Monthly Channel
 - Semi-annual Channel
 
-For more details on the Office 2016 release cadence names see [Slow - Fast Level Names](https://support.office.com/en-US/article/Changes-to-Slow-Fast-level-names-for-Office-Insider-for-Windows-desktop-055ee4f9-9ce3-4fb8-8a9a-ca6745867d52).
+For more information about the Office 2016 release cadence names, see [Slow - Fast Level Names](https://support.office.com/en-US/article/Changes-to-Slow-Fast-level-names-for-Office-Insider-for-Windows-desktop-055ee4f9-9ce3-4fb8-8a9a-ca6745867d52).
 
 The following sections discuss some of the features that have been introduced in Excel 2016 that you can use to improve performance with large or complex workbooks.
 
 ### Large Address Aware (LAA) Memory improvement for 32-bit Excel
-Although 64-bit Excel has extremely large virtual memory limits, 32-bit Excel has been limited to 2 Gigabytes (GB). And many Excel customers have found it difficult to migrate to 64-bit Excel because they use third-party addins and controls that are not available in 64-bit versions.
-LAA has now been enabled for 32-bit versions of Excel 2013 and Excel 2016, and will minimize out of memory error messages.
-LAA doubles available virtual memory from 2GB to 4GB when using 64-bit Windows, and increases available virtual memory from 2GB to 3GB under 32-bit Windows.
+Although 64-bit Excel has extremely large virtual memory limits, 32-bit Excel has been limited to 2 gigabytes (GB). Many Excel customers have found it difficult to migrate to 64-bit Excel because they use third-party add-ins and controls that are not available in 64-bit versions.
```

Comment:
To help keep headings more concise, let's use acronyms there and spell out in the text.

Reply:
Can we smooth out the text a bit here - rewrite in active voice, and change the tone to be a bit less negative. Suggest "Although the 64-bit version of Excel has large virtual memory limits, the 32-bit version has only 2 GBs of virtual memory. Some customers use the 32-bit version because some third-party add-ins and controls are not available in the 64-bit version."